source/onboard/migrating-to-mattermost.rst

```diff
@@ -120,12 +120,89 @@ Migrating from Slack Using the Mattermost mmetl Tool and Bulk Import
   
   This method is the recommended way to import Slack's corporate export file. It can be used with Mattermost v5.0 and onwards.
 
-1. Use the `slack-advanced-exporter <https://github.com/icelander/slack-advanced-exporter>`_ to download attachments and add users' email addresses to your Slack corporate export file.
-2. Use the `mmetl tool <https://github.com/mattermost/mmetl>`_ to transform Slack's corporate export file into the ``jsonl`` format required by the bulk import tool.
-3. Bulk load the files using the steps provided in the `bulk loading documentation <https://docs.mattermost.com/deployment/bulk-loading.html#bulk-loading-data>`_.
+**1. Prepare your Mattermost Server**
+
+We recommend you create a new team in Mattermost to hold the imported Slack data. You can import this into an existing team, but ensure there are no channel name collisions. Also, make sure that all users in Mattermost have the same username as in Slack, otherwise the import will fail. Also, system administrator roles will be overwritten if the usernames match and the user isn't an admin on the Slack workspace.
+
+**2. Generate a Slack import**
+
+The first step is to generate a `Slack export <https://slack.com/help/articles/201658943-Export-your-workspace-data>`_.
+
+Next, follow these steps to create a bot token:
+
+1. Go to https://api.slack.com/apps.
+2. Select **Create New App**.
+3. Select **From scratch**.
+4. Name the app something like "Slack Advanced Exporter" and select the workspace. You'll have to do this for every workspace. Then create the app.
+5. Select **OAuth & Permissions** and scroll down to **Scopes**.
+6. Under **Bot Token Scopes** select ``users:read`` and ``users:read.email``.
+7. Scroll up and select **Install to Workspace**.
+8. Grant the app permissions.
+9. Copy the Bot User OAuth Token and save it somewhere convenient.
+
+**3. Download file attachments and email addresses**
+
+The Slack export does not include file attachments and email addresses, so you must use ``slack-advanced-exporter`` to download them. Download the latest release of ``slack-advanced-exporter`` for your OS and architecture `here <https://github.com/icelander/slack-advanced-exporter/releases/>`__ and extract it.
```

Comment:
If we reference Github repositories with wouldn't it be better if they life in the Mattermost namespace?

Reply:
Yes, it definitely would be better. I guess one question is whether this is officially supported or not - if not, then it can stay where it is imo.

Reply:
@justinegeffen - Any outstanding changes needed for this PR, or can it be merged into master and promoted to Production?